java/client/src/main/java/glide/api/logging/Logger.java

```diff
@@ -140,7 +138,7 @@ public static void log(
             initLogger(Level.DEFAULT, null);
         }
 
-        if (level == Level.DISABLED) {
+        if (level == Level.OFF) {
```

Comment:
see my comment above

Reply:
As said, it was already there as you see, but even if not, answered above.